operators/olm-webhooks.adoc

```diff
@@ -0,0 +1,19 @@
+[id="olm-webhooks"]
+= Managing admission webhooks in Operator Lifecycle Manager
+include::modules/common-attributes.adoc[]
+:context: olm-webhooks
+
+toc::[]
+
+Validating and mutating admission webhooks allow Operator authors to intercept,
+modify, and accept or reject resources before they are handled by the Operator
```

Comment:
Technically the resource in intercepted before the resource is saved to etcd.

Reply:
Follow-up: https://github.com/openshift/openshift-docs/pull/23530